merchant-app/app/src/main/java/com/googleinterns/smb/common/BitmapUtils.java

```diff
@@ -31,15 +31,13 @@ public static Bitmap getBitmap(ByteBuffer data, FrameMetadata metadata) {
             YuvImage image =
                     new YuvImage(
                             imageInBuffer, ImageFormat.NV21, metadata.getWidth(), metadata.getHeight(), null);
-            if (image != null) {
```

Comment:
Any specific reasoning to remove the null check

Reply:
image is always a new object, so will never be null